libusb/core.c

```diff
@@ -116,15 +116,17 @@ struct list_head active_contexts_list;
  * libusb uses stderr for all logging. By default, logging is set to NONE,
  * which means that no output will be produced. However, unless the library
  * has been compiled with logging disabled, then any application calls to
- * libusb_set_debug(), or the setting of the environmental variable
- * LIBUSB_DEBUG outside of the application, can result in logging being
- * produced. Your application should therefore not close stderr, but instead
- * direct it to the null device if its output is undesirable.
- *
- * The libusb_set_debug() function can be used to enable logging of certain
- * messages. Under standard configuration, libusb doesn't really log much
- * so you are advised to use this function to enable all error/warning/
- * informational messages. It will help debug problems with your software.
+ * libusb_set_option(ctx, LIBUSB_OPTION_LOG_LEVEL), or the setting of the
```

Comment:
The log level is missing the call ``libusb_set_option(ctx, LIBUSB_OPTION_LOG_LEVEL)``.
I guess it should be ``libusb_set_option(ctx, LIBUSB_OPTION_LOG_LEVEL, log_level)`` instead.

The same problem is also present many times in other comments.